data/json/monsters/feral_humans.json

```diff
@@ -215,5 +215,135 @@
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 80,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "@",
+    "color": "magenta",
+    "aggression": 75,
+    "morale": 100,
+    "melee_skill": 3,
+    "melee_dice": 3,
+    "melee_dice_sides": 6,
+    "melee_cut": 5,
+    "dodge": 4,
+    "armor_bash": 7,
+    "armor_cut": 7,
+    "armor_bullet": 6,
+    "armor_acid": 4,
+    "vision_day": 50,
+    "vision_night": 3,
+    "harvest": "human",
+    "upgrades": { "into": "mon_feral_survivalist", "half_life": 120 },
+    "starting_ammo": { "9mm": 3 },
+    "special_attacks": [
+      [ "GRAB", 7 ],
+      {
+        "type": "gun",
+        "cooldown": 10,
+        "move_cost": 50,
+        "gun_type": "m9",
+        "ammo_type": "9mm",
+        "fake_skills": [ [ "gun", 1 ], [ "pistol", 1 ] ],
+        "fake_dex": 8,
+        "fake_per": 9,
+        "ranges": [ [ 0, 14, "DEFAULT" ], [ 10, 35, "DEFAULT" ] ],
+        "require_targeting_player": false,
+        "description": "The feral prepper haphazardly fires their firearm!"
+      }
+    ],
+    "death_drops": "mon_feral_prepper_death_drops",
+    "zombify_into": "mon_zombie_survivor",
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "HUMAN",
+      "CAN_OPEN_DOORS",
+      "CLIMBS",
+      "PUSH_MON",
+      "PATH_AVOID_DANGER_1",
+      "DROPS_AMMO"
+    ]
+  },
+  {
+    "id": "mon_feral_survivalist",
+    "type": "MONSTER",
+    "name": { "str": "feral survivalist" },
+    "description": "Using formidable equipment, this feral prepper is seen having an athletic body, a sign of sturdy training before they turned feral.  In addition of handling their firearm more properly, they seem to have an awareness of self preservation when in critical harm, a byproduct of leftover survival instincts kicking in.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 100,
+    "speed": 110,
+    "material": [ "flesh" ],
+    "symbol": "@",
+    "color": "magenta",
+    "aggression": 75,
+    "morale": 12,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 3,
+    "melee_cut": 5,
+    "dodge": 4,
+    "armor_stab": 18,
+    "armor_bash": 15,
+    "armor_cut": 25,
+    "armor_bullet": 20,
+    "armor_acid": 4,
+    "vision_day": 50,
+    "vision_night": 6,
+    "harvest": "human",
+    "starting_ammo": { "9mm": 6 },
+    "special_attacks": [
+      [ "GRAB", 7 ],
+      {
+        "type": "gun",
+        "cooldown": 5,
+        "move_cost": 150,
```

Comment:
Why a higher move cost than the prepper?

Reply:
I remember that Malecypse left a review on the prepper's movecost when it was 150, saying  something that goes: "This means that the feral prepper take 1.5 seconds to aim". But then following the design that the feral prepper treats the gun more like a melee weapon, they forget how to aim, and just pull the trigger (something like that). So I figured maybe it will make sense they only take half a second to aim, since they technically don't aim.
But maybe I should make it back to 150? What would you suggest?

Reply:
Oh, its supposed to represent the difference between a madman just shooting randomly and one that actually takes its time to aim, then its all good! It would help to leave a comment explaining the rationale so someone in the future doesn't change it if they see it in contrast with the prepper.

Reply:
ohh like a "//": ? alright will do thank you